src/util/compatibility/qatomic.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
+
+#include <QAtomicInteger>
+#include <QAtomicPointer>
+
+template<typename T>
+inline T atomicLoadAcquire(const QAtomicInteger<T>& atomicInt) {
+    // TODO: QBasicAtomicInteger<T>::load() is deprecated and should be
+    // replaced with QBasicAtomicInteger<T>::loadRelaxed() However, the
+    // proposed alternative has just been introduced in Qt 5.14. Until the
+    // minimum required Qt version of Mixxx is increased, we need a version
+    // check here
```

Comment:
Does this comment really need to be repeated almost verbatim 6 times? I think one comment at the top of the file would be sufficient.